bambi/priors.py

```diff
@@ -59,6 +59,11 @@ def update(self, **kwargs):
         Args:
             kwargs (dict): Optional keyword arguments to add to prior args.
         '''
+        # Backends expect numpy arrays, so make sure all numeric values are
+        # represented as such.
+        for k, v in kwargs.items():
```

Comment:
You don't need the outer for loop here, right? Just the dictionary comprehension?